engine/src/java/com/runrev/android/DialogModule.java

```diff
@@ -149,7 +149,7 @@ public void onDismiss(DialogInterface p_dialog)
         Calendar t_calendar = Calendar.getInstance();
         t_calendar.setTimeInMillis(p_current * 1000);
         
-        DatePickerDialog t_dialog = new DatePickerDialog(m_engine.getContext(), t_date_listener,
+        DatePickerDialog t_dialog = new DatePickerDialog(m_engine.getContext(), android.R.style.Theme_DeviceDefault_Light_Dialog_Alert, t_date_listener,
```

Comment:
Cool @livecodepanos . Is there a reason not to use `Theme_DeviceDefault_Dialog_Alert` as in do we need to force `Light`?

Reply:
Hehe, the reason for that was that I did not notice there was such an option :)

Will update the PR asap